controller.go

```diff
@@ -778,6 +778,8 @@ func (c *controller) addNetwork(n *network) error {
 		return err
 	}
 
+	n.startResolver()
+
```

Comment:
A container connected to multiple networks should be able to resolve the service names in any network its connected to. This is done by walking through all the end-points in `resolveName` and lookup the service DB of the network to which the EP belongs.

For Windows containers if the query from the container can be associated with only one of the networks (depending on which interface the client sends the query on) is there any way to search the other networks as well ?